docs/_pages/0131-dev-env.md

```diff
@@ -410,6 +410,34 @@ To check **VSecM Safe**’s logs, execute the following:
 kubectl logs -n vsecm-system $NAME_OF_VSECM_SAFE_POD -f 
 ```
 
+## Inspecting SPIRE Server via CLI
+
```

Comment:
SPIRE server has a decent CLI.

When there is a problem, it’s worthwhile using SPIRE Server CLI too.